example/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
I would use the same license as for the project here :) https://github.com/asyncapi/bundler/blob/d5ece85b1b18921f252228aba88c4aafe7884196/package.json#L23